VERSION

```diff
@@ -1 +1 @@
-20.1.7
+20.2.2
```

Comment:
The version is right? I wasn't entirely sure because the old version is `20.1.7` but the `ERLANG_VERSION` in the Dockerfile was `20.2.1`